bentoml/yatai/yatai_service_impl.py

```diff
@@ -398,7 +396,11 @@ def GetBento(self, request, context=None):
 
             if bento_pb:
                 if bento_pb.uri.type == BentoUri.S3:
-                    bento_pb.uri.s3_presigned_url = self.repo.get(
+                    bento_pb.uri.cloud_presigned_url = self.repo.get(
+                        bento_pb.name, bento_pb.version
+                    )
+                if bento_pb.uri.type == BentoUri.GCS:
+                    bento_pb.uri.cloud_presigned_url = self.repo.get(
                         bento_pb.name, bento_pb.version
                     )
```

Comment:
Let's refactor these two.

Reply:
@yubozhao As far as I can understand, we should merge these 2 if conditions. Please confirm.

Reply:
Yes we should refactor these two if statement.

Reply:
Got it.